staff/marti-malek/mailchimp/index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Mailchimp</title>
+    <link rel="stylesheet" href="./styles/styles.css">
+</head>
+<body>
+    <main>
+        <header class="header">
+            <div class="header__log"><p>Log In</p></div>
+            <div class="header__nav">
+                <img src=".\styles\main_logo.png" alt="logo">
+                <button class="button">Sign Up Free</button>
+                <nav>
+                    <a href="">Pricing</a>
+                    <a href="">Features</a>
+                    <a href="">Resources</a>
+                    <a href="">Support</a>
+                    <a href="">Blog</a>
+                    <input type="text" placeholder="Search">
+                    <img src=".\styles\search.png" alt="search">
+                </nav>
+            </div>
+        </header>
+        <section class="first">
+            <img class="first__monkey" src=".\styles\easy-email-newsletters-banner-png8.png" alt="chimp">
+            <img class="first__title" src=".\styles\easy-email-newsletters-text.png" alt="">
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. A vel minus maiores maxime quas, facere reiciendis architecto. Ad pariatur, atque praesentium adipisci tempora similique minima provident quidem corporis molestias tenetur.</p>
+            <button class="button">Sign Up Free</button>
+        </section>
+        <section class="center">
+            <div class="container">
+                <h3>What's New</h3>
+                <section class="one">
+                    <img src=".\styles\templates.jpg" alt="">
```

Comment:
why you use a inverted \

Reply:
Windows copies folder paths with this by default.